openy_af4_vue_app/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "vue-cli-service build --target lib --formats umd-min --name activity_finder_4 src/main.js",
-    "dev": "vue-cli-service build --target lib --formats umd-min --name activity_finder_4 src/main.js --watch",
+    "dev": "vue-cli-service build --target lib --formats umd-min --name activity_finder_4 src/main.js --watch --mode development",
```

Comment:
This sets development mode so that Vue dev tools can be used. Tools are enabled conditionally based on the environment variable.